rpc/src/main/scala/client/handlers/ClientCallsMHandler.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.rpc
+package client.handlers
+
+import freestyle.Capture
+import freestyle.async.AsyncContext
+import freestyle.rpc.client.ClientCallsM
+import io.grpc.{CallOptions, Channel, ClientCall, MethodDescriptor}
+import io.grpc.stub.{ClientCalls, StreamObserver}
+import freestyle.rpc.client.implicits._
+
+import scala.collection.JavaConverters._
+
+class ClientCallsMHandler[F[_]](implicit C: Capture[F], AC: AsyncContext[F])
```

Comment:
We have traditionally been parametrizing handlers with `M[_]` instead of `F[_]` since in other classes aggregating algebras `F[_]` we have used it to refer to the coproduct. I'm fine if we call this F but in that case we should do this across all modules. Also it would be nice if we all as a team come up with a naming convention doc that we enforce across the entire org to make sure we are consistent and not confusing users.